### DISCUSSION TITLE
How to customize zooming shortcuts?

### DISCUSSION THREAD

**Discussion by volt4ire** [vslinko/obsidian-zoom#15] [Q&A] [answered]
I currently already have `ctrl [shift] .` assigned to other shortcuts, so was wondering if you could add an option to customize the zooming shortcuts. Thanks!

**Comment by FelipeRearden** [comment, marked as answer]
@volt4ire 

Obsidian has a bulit-in feature to change the plug-ins hotkeys. Look:
![Screen Shot 2021-06-10 at 12 51 39](https://user-images.githubusercontent.com/85180696/121557338-0e56d000-c9eb-11eb-953a-eba3c9ba1a8e.png)

- Open the Menu: HotKeys
- Write Zoom 
- Change the Shortcut with the latest button 

You can access all the commands with the Command Palette
![Screen Shot 2021-06-10 at 12 57 03](https://user-images.githubusercontent.com/85180696/121558049-aeacf480-c9eb-11eb-9db4-a05e1da025d4.png)

![Screen Shot 2021-06-10 at 12 58 14](https://user-images.githubusercontent.com/85180696/121558077-b5d40280-c9eb-11eb-9852-39a06b03173e.png)


I hope it helps!

Have a great day!

**Reply by volt4ire** [reply]
Awesome thanks so much!!

**Reply by FelipeRearden** [reply]
I'm glad you liked! Have a great day.
